bin/nrdebug

```diff
@@ -212,7 +212,7 @@ class ProcessReport
 
       section(f) do
         c_backtraces, ruby_backtraces = @target.gather_backtraces
-        if c_backtraces.match(/could not attach/i)
+        if /could not attach/i.match?(c_backtraces)
```

Comment:
Hmm... I agree that `#match?` is a much better and more performant fit for what we're trying to do here (simply "does it match", not "give me the matching content"), but I don't like the re-arranging of the components and calling `#match` on the `i` of the regex pattern.

I'd prefer this if it still passes the checks:

```ruby
if c_backtraces.match?(/could not attach/i)
```